qa/setup_dali_extra.sh

```diff
@@ -2,7 +2,7 @@
 
 # Fetch test data
 export DALI_EXTRA_PATH=${DALI_EXTRA_PATH:-/opt/dali_extra}
```

Comment:
After you bump the commit in DALI_EXTRA_VERSION you can revert the changes in this file.